linetools/isgm/utils.py

```diff
@@ -402,7 +402,7 @@ def table_from_complist(complist, summed_ion=False, NHI_obj=None, vrange=None,
     tab['ion'] = [icomp.Zion[1] for icomp in complist]
 
     # . attributes (required ones)
-    for attrib in ['zcomp', 'Ej', 'flag_N', 'logN', 'sig_logN',
+    for attrib in ['zcomp', 'Ej',
```

Comment:
Should likely remove `b`, `sig_b`, `vel` and `sig_vel` too

Reply:
Agreed. The idea now is to impose all these attributes to go in the `.attrib` dict()